apple/testing/default_runner/ios_xctestrun_runner.bzl

```diff
@@ -120,6 +129,12 @@ will always use `xcodebuild test-without-building` to run the test bundle.
             default = True,
             doc = """
 Toggle simulator reuse. The default behavior is to reuse an existing device of the same type and OS version. When disabled, a new simulator is created before testing starts and shutdown when the runner completes.
+""",
+        ),
+        "build_for_device": attr.bool(
+            default = False,
+            doc = """
+If enabled then the runner will support testing on a physical device.
```

Comment:
Is it possible to package up this functionality while avoiding a separate runner for a device vs. simulator? Thinking something like the previous runner (but mapping it to this one's input params):

https://github.com/bazelbuild/rules_apple/blob/79f4d28f052213faea134901806c48f8a175e079/apple/testing/default_runner/ios_test_runner.template.sh#L226-L231

Reply:
That looks good! Will do that instead